packages/braid-design-system/src/lib/components/Autosuggest/Autosuggest.docs.tsx

```diff
@@ -474,11 +473,12 @@ const docs: ComponentDocs = {
           each suggestion.
         </Text>
       ),
-      Example: ({ id, setDefaultState, setState, getState }) =>
-        source(
+      Example: ({ id, setDefaultState, setState, getState }) => {
+        const suggestion = 'Apples and Bananas';
```

Comment:
I would suggest reverting this to default state as the content of passed into `source` is used to generate the Playroom snippet, im guessing opening this example in Playroom errors with `suggestion` being undefined?

Reply:
It would error, but we hide the code and option to open in Playroom since it doesn't actually use Autosuggest. But yeah happy to revert

Reply:
Ah good shout! No probs